internal/app/components/users/users.go

```diff
@@ -89,3 +90,8 @@ func HashPassword(password string) (PasswordHash, error) {
 	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
 	return PasswordHash(bytes), err
 }
+
+// Creates a new timemline with the courses added to the course CourseHistory
+func (usr User) UpdateTimeline(name TimelineName) {
```

Comment:
can we name this `NewTimeline`? `UpdateTimeline` is kinda confusing because it's actually adding a new one instead of updating

Reply:
Will change this to NewTimeline